docs/Testing-Releases.md

```diff
@@ -1,16 +1,15 @@
 # Testing FHIR Server for Azure Releases
 
 ## Description
+This guide provides information to deploy OSS FHIR Server with latest release. This provides you the flexibility to run your own integration test suite, against the latest OSS version of FHIR API.
 
-This guide provides information to set up a docker container with the latest release of FHIR Server for Azure so that you can run your own integration tests against the API.
+## Understanding the changes in the release
+To understand the changes shipped as part of a particular OSS release. Refer to the content under  [releases](https://github.com/microsoft/fhir-server/releases) section in Github.
 
-## Why
-
-FHIR Server for Azure works on a release cadence of around every two weeks. You can find information detailing the technical content of the releases under the [releases](https://github.com/microsoft/fhir-server/releases) section in Github.
-
-After a release has been created, these assets will be staged in the next deployment of [Azure API for FHIR](https://azure.microsoft.com/en-us/services/azure-api-for-fhir/). The deployments for Azure API for FHIR will usually roll out the following week after the OSS release, giving your team about a week to test with these incremental releases.
+After an OSS release has been created, these assets will be staged in the next deployment of Azure release. The deployments for FHIR Service will usually roll out the following week after the OSS release.
```

Comment:
Since we release on Fridays, it was a little ambiguous to me if the intended meaning is "the very next week", or "OSS Release +1 week"